views/snippets/ajax_post.html

```diff
@@ -0,0 +1,27 @@
+<script src="jquery.js"></script>
+<script>
+  $.ajax({
+    type: 'POST',
+    url: 'http://localhost:8080/',
```

Comment:
Can we use `https://jsonp.afeld.me` here, to be consistent with the other examples?